README.md

```diff
@@ -26,7 +26,7 @@ If you are interested in contributing to EvalAI-CLI, follow our [contribution gu
     git clone https://github.com/Cloud-CV/evalai-cli.git evalai-cli
     ```
 
-3. Create a virtual environment
+3. Create a virtual environment ([Mac users could easily use Homebrew](https://gist.github.com/apavamontri/4516816))
```

Comment:
Instead of just for Mac users, can you add virtual env installation link to a page/post/steps that includes linux and windows as well? Something like: https://pythonforundergradengineers.com/virtualenv-in-osx-linux-windows.html